### DISCUSSION TITLE
Remove snippet section delimiter from whole file

### DISCUSSION THREAD

**Discussion by joapuiib** [facelessuser/pymdown-extensions#2540] [Q&A] [answered]
Hi, I just found [Snippet Sections](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippet-sections) and I love it so far.

```python
# --8<-- [start:func]
def my_function(var):
    pass
# --8<-- [start:end]
```

I'm using for showing some portions of a python file, but I would want to add
the whole Python script at the end of the page.

Is there any way to not include snippet section delimiters?
```
--8<-- "regressio_metriques.py"
```

I would want to remove all `# --8<--` lines.

![image](https://github.com/user-attachments/assets/63d3c0f3-5e23-44c9-84e2-833d65db7212)

**Comment by facelessuser** [comment]
I'm pretty sure sections should get auto-removed. Is that now what you are seeing?

**Reply by joapuiib** [reply]
Section delimiters are removed when using snippet sections:

- https://joapuiib.github.io/iabd-saa/apunts/04_regressio/01_regressio_lineal/#error-absolut-mitja-mae

````markdown
```python
--8<-- "regressio_metriques.py:mae_pandas"
```
````

![image](https://github.com/user-attachments/assets/616fbf9f-1c97-441d-96a4-5e77d52c92b5)

But, they are present when including the **whole file**.

- https://joapuiib.github.io/iabd-saa/apunts/04_regressio/01_regressio_lineal/#codi-font

````markdown
```python
--8<-- "regressio_metriques.py"
```
````

![image](https://github.com/user-attachments/assets/0bebc32e-4364-4c5f-88d5-c7cd8b24bcee)

**Reply by facelessuser** [reply]
Please create an issue so I can take a closer look.

**Reply by facelessuser** [reply, marked as answer]
Ended up having some time and created a PR: https://github.com/facelessuser/pymdown-extensions/pull/2541

**Reply by joapuiib** [reply]
Section delimiters are no longer included with this PR.

![image](https://github.com/user-attachments/assets/80bd67c0-73b9-43d3-9c24-6d04e5f9e654)

Thanks for taking the time!
